### PR TITLE
make request duration useful

### PR DESCRIPTION
**Pull request by jrwren**
Use mux path pattern instead of url path.
Use seconds instead of milliseconds.
Include v4/v5 prefix.

### CODE REVIEW

File: internal/router/router.go [hunk 1]
```diff
@@ -836,6 +836,11 @@ func (r *Router) PutMetadata(id *ResolvedURL, data map[string]*json.RawMessage,
 	return nil
 }
 
+func (r *Router) Pattern(req *http.Request) (pattern string) {
```
Comment (rogpeppe) [inline internal/router/router.go]: doc comment and test please.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1896/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1897/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1898/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1899/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1900/
Test PASSed.

**Comment by mhilton** [comment]
:+1:

**Comment by bac** [comment]
🍣

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**